src/components/Breed/ImagesGallery.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+import classes from "./ImagesGallery.module.css";
+
+const ImageGallery = (props) => {
```

Comment:
you can destructure the properties you are goin go use. Something like this 
```tsx
const ImageGallery = ({imgUrl, imgOnload}) => {
   .....
}
```
then you can directly use `imgUrl` instead of `props.imgUrl`

Reply:
This is done